components/textbox/src/_mixin.scss

```diff
@@ -291,13 +306,85 @@ $utrecht-support-prince-xml: false !default;
   --utrecht-textbox-value-max-length: 4;
 }
 
+@mixin utrecht-textbox--voorletters-size {
+  /* "Voorletters" depends on how long your name is and after research of the longest name in the Netherlands it can be about 8 to 9 characters.
+     For now im using the longest name source suggestion: 8 */
+  --utrecht-textbox-value-max-length: 8;
+}
+
 @mixin utrecht-textbox--voorvoegsel-size {
   /* "Tabel 36 Voorvoegselstabel" by "Rijksdienst voor Identiteitsgegevens" has a list
    * of all allowed values. The longest value is 10 characters.
    * The class name is in Dutch because it is a Dutch concept. */
   --utrecht-textbox-value-max-length: 10;
 }
 
+@mixin utrecht-textbox--voornamen-size {
+  /* Longest name in the Netherlands is 50 characters. Suggesting: 40 */
+  --utrecht-textbox-value-max-length: 40;
+}
+
+@mixin utrecht-textbox--vestigingnummer-size {
+  /* Typically 9 or lower digits */
+  --utrecht-textbox-value-max-length: 10;
+}
+
+@mixin utrecht-textbox--bsn-size {
+  /* BSN is 8 or 9 characters. Suggest: 5 */
+  --utrecht-textbox-value-max-length: 5;
```

Comment:
Kunnen we niet beter voor 9 gaan dan?
We kunnen eventueel de multiplier aanpassen als het om digits gaat, bijv:

```
  --_utrecht-textbox-value-char: 1ch;
```

Misschien kunnen we het wel reusable aanpakken:

```scss
@mixin utrecht-textbox--numeric {
  --_utrecht-textbox-value-char: 1ch;
  -moz-appearance: textfield; /* avoid spinner input in Firefox */
  font-variant-numeric: lining-nums tabular-nums;
}
```

en dan:

```scss
@mixin utrecht-textbox--bsn-size {
  @include utrecht-textbox--numeric;
  
  /* BSN is 8 or 9 characters */
  --utrecht-textbox-value-max-length: 9;
}
```